src/handlers/comments.ts

```diff
@@ -74,15 +77,15 @@ export function createKey(issueUrl: string, issue?: number) {
  */
 export function streamlineComments(comments: SimplifiedComment[]) {
   const streamlined: Record<string, StreamlinedComment[]> = {};
+
   for (const comment of comments) {
     const { user, issueUrl: url, body } = comment;
-    // Skip bot comments
     if (user?.type === "Bot") continue;
+
     const key = createKey(url);
     const [owner, repo] = splitKey(key);
-    if (!streamlined[key]) {
-      streamlined[key] = [];
-    }
+    streamlined[key] ??= [];
```

Comment:
??=

This is a new one I've never seen before. Pretty nifty

Reply:
I've used in a few of my plugins now and yeah far more readable